samples/ts/ts-customer-registry/package.json

```
@@ -0,0 +1,60 @@
+{
+  "name": "ts-customer-registry",
+  "version": "0.0.0",
+  "description": "Akka Serverless JavaScript SDK sample application for customer registry",
```

Shall we call it the `TypeScript SDK` or have it mentioned as a TypeScript sample in some way?

```suggestion
  "description": "Akka Serverless TypeScript SDK sample application for customer registry",
```